PersistedClusterStateServiceTests.testSlowLogging assertion fails reproduceably

**Build scan**: [Most recent failure](https://gradle-enterprise.elastic.co/s/iya4umiiny4fe)

**Repro line**: 
```
./gradlew ':server:test' --tests "org.elasticsearch.gateway.PersistedClusterStateServiceTests.testSlowLogging" -Dtests.seed=ED8BECBD5297494D -Dtests.security.manager=true -Dtests.locale=sk-SK -Dtests.timezone=NZ -Druntime.java=8
```

**Reproduces locally?**: Yes, fairly reliably

**Applicable branches**: 7.x

**Failure history**: Looks to be [failing once or twice a month](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-6M,mode:quick,to:now))&_a=(columns:!(_source),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:e58bf320-7efd-11e8-bf69-63c8ef516157,key:project,negate:!f,params:(query:elasticsearch,type:phrase),type:phrase,value:elasticsearch),query:(match:(project:(query:elasticsearch,type:phrase)))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:e58bf320-7efd-11e8-bf69-63c8ef516157,key:test,negate:!f,params:(query:testSlowLogging,type:phrase),type:phrase,value:testSlowLogging),query:(match:(test:(query:testSlowLogging,type:phrase)))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:e58bf320-7efd-11e8-bf69-63c8ef516157,key:message,negate:!f,params:(query:'java.lang.AssertionError:%20max%20must%20be%20%3E%3D%20min:%201,%200',type:phrase),type:phrase,value:'java.lang.AssertionError:%20max%20must%20be%20%3E%3D%20min:%201,%200'),query:(match:(message:(query:'java.lang.AssertionError:%20max%20must%20be%20%3E%3D%20min:%201,%200',type:phrase))))),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'class:%22org.elasticsearch.gateway.PersistedClusterStateServiceTests%22'),sort:!(time,desc))) for the last 6 months
<!--
Link to build stats and possible indication of when this started failing and how often it fails
<https://build-stats.elastic.co/app/kibana>
-->
**Failure excerpt**:
```
org.elasticsearch.gateway.PersistedClusterStateServiceTests > testSlowLogging FAILED |  
-- | --
  | java.lang.AssertionError: max must be >= min: 1, 0 |  
  | at __randomizedtesting.SeedInfo.seed([ED8BECBD5297494D:5EA64A62FB65353F]:0) |  
  | at com.carrotsearch.randomizedtesting.generators.RandomNumbers.randomLongBetween(RandomNumbers.java:29) |  
  | at org.elasticsearch.test.ESTestCase.randomLongBetween(ESTestCase.java:586) |  
  | at org.elasticsearch.gateway.PersistedClusterStateServiceTests.testSlowLogging(PersistedClusterStateServiceTests.java:861)
```

--- a/server/src/test/java/org/elasticsearch/gateway/PersistedClusterStateServiceTests.java
+++ b/server/src/test/java/org/elasticsearch/gateway/PersistedClusterStateServiceTests.java
@@ -856,7 +856,7 @@ public class PersistedClusterStateServiceTests extends ESTestCase {
                     "writing cluster state took [*] which is above the warn threshold of [*]; " +
                         "wrote global metadata [false] and metadata for [1] indices and skipped [0] unchanged indices"));
 
-                writeDurationMillis.set(randomLongBetween(1, writeDurationMillis.get() - 1));
+                writeDurationMillis.set(randomLongBetween(0, writeDurationMillis.get() - 1));
                 assertExpectedLogs(1L, clusterState, newClusterState, writer, new MockLogAppender.UnseenEventExpectation(
                     "should not see warning below threshold",
                     PersistedClusterStateService.class.getCanonicalName(),

